test/entrypoint.test.ts

```diff
@@ -275,14 +276,18 @@ describe('EntryPoint', function () {
         // we need maxFeeperGas > block.basefee + maxPriorityFeePerGas so requiredPrefund onchain is basefee + maxPriorityFeePerGas
         maxFeePerGas: block.baseFeePerGas.mul(3),
```

Comment:
actually, this field looks anachronism: why doesn't it use DefaultUserOp or fillUserOp ?

Reply:
I didn't touch this code at all